### ISSUE TITLE
Error when following installation steps in doc

### ISSUE THREAD

**Issue by unarmedcivilian** [metacontroller/metacontroller#611]
Attempting to install using the instructions in the [doc](https://metacontroller.github.io/metacontroller/guide/install.html#install-metacontroller-using-kustomize) leads to this error 

```
Error response from daemon: manifest for metacontrollerio/metacontroller:v4.3.4 not found: manifest unknown: manifest unknown
```
In dockerhub I see the image tag is actually `4.3.4` rather than `v4.3.4`. I edited the [maifest](https://github.com/metacontroller/metacontroller/blob/master/manifests/production/metacontroller.yaml#L23) file and got it working.

**Comment by grzesuav** [comment]
Reopening, have some issues with releasing new version

**Comment by grzesuav** [comment]
Fixed in 4.3.5
